src/PowerShellEditorServices/Properties/AssemblyInfo.cs

```diff
@@ -5,6 +5,7 @@
 
 using System.Runtime.CompilerServices;
 
+[assembly: InternalsVisibleTo("Microsoft.PowerShell.EditorServices.Protocol")]
```

Comment:
About time 😁 

Might be worth looking into public key signed strong names.

Reply:
It also turns out we [can add this in the csproj](https://stackoverflow.com/a/49978185) instead if we generate the `AssemblyInfo.cs`. Given that these already exist, I think the best solution is to beef up all the `AssemblyInfo.cs` files with `InternalsVisibleTo` directives for all the PSES assemblies